JetStreamDriver.js

```diff
@@ -2248,6 +2248,21 @@ let BENCHMARKS = [
         iterations: 40,
         tags: ["Wasm"],
     }),
+    // nutrient
+    new WasmEMCCBenchmark({
+        name: "nutrient-wasm",
+        files: [
+            "./wasm/nutrient/helper.js",
+            "./wasm/nutrient/build/nutrient-viewer.wasm.js",
+            "./wasm/nutrient/benchmark.js",
+        ],
+        preload: {
+            pdfDocument: "./wasm/nutrient/assets/example.pdf",
+            wasmBinary: "./wasm/nutrient/build/nutrient-viewer.wasm",
+        },
+        iterations: 10,
```

Comment:
I think we can easily increase that to 40 iterations, which gives us potentially less noisy measurement and less incentive to tier-up super aggressively on short running workloads. (On an x64 workstation, even with 40 iterations we are still under 2s Wall time, which is more than fine.)